### ISSUE TITLE
What is the top 5 accuracy u got in lfw dataset

### ISSUE THREAD

**Issue by RUPESHKUMARPALLAI** [ZhaoJ9014/face.evoLVe.PyTorch#161]
I got a top5 accuracy of about 85% in lfw training dataset what is the expected one?
